### DISCUSSION TITLE
Is this documentation still correct?

### DISCUSSION THREAD

**Discussion by ghost** [gajus/slonik#491] [General]
This page: https://github.com/gajus/slonik/blob/main/.README/QUERY_BUILDING.md

Directs the reader to read this:

https://dev.to/gajus/dynamically-generating-sql-queries-using-node-js-2c1g

Are both pages still correct and up to date with the latest in how slonik works?

**Comment by TmLev** [comment]
Seems like the article is more than 4 years old, so I'd assume it's not relevant anymore.
